controllers/profiles/reconciler_prod_test.go

```diff
@@ -39,15 +41,16 @@ func Test_reconcilerProdBuildConditions(t *testing.T) {
 	platform := test.GetKogitoServerlessPlatformInReadyPhase("../../config/samples/"+test.KogitoServerlessPlatformWithCacheYamlCR, t.Name())
 	client := test.NewKogitoClientBuilder().WithRuntimeObjects(workflow, platform).Build()
 
-	result, err := NewReconciler(client, &logger, workflow).Reconcile(context.TODO(), workflow)
+	config := &rest.Config{}
+	result, err := NewReconciler(client, config, &logger, workflow).Reconcile(context.TODO(), workflow)
```

Comment:
maybe rename the NewReconciler to Reconciler only?

Reply:
Also for this See https://github.com/kiegroup/kogito-serverless-operator/pull/78#discussion_r1152960513